src/omnisharpMain.ts

```diff
@@ -91,8 +91,12 @@ export function activate(context: vscode.ExtensionContext): any {
 		server.stop();
 	}));
     
+	// register empty handler for csharp.installDebugger
+	// running the command activates the extension, which is all we need for installation to kickoff
+	disposables.push(vscode.commands.registerCommand('csharp.downloadDebugger', () => { }));
```

Comment:
We registered all the other commands in src\features\commands.ts. Should that be there also?